src/world/terrain/chunk.cpp

```diff
@@ -3,51 +3,219 @@
 #include "material.hpp"
 #include "path/tile_iterators.hpp"
 #include "terrain.hpp"
-#include "terrain_base.hpp"
 #include "tile.hpp"
 
 namespace terrain {
 
 Chunk::Chunk(TerrainDim3 chunk_position, Terrain* ter) :
-    ter_(ter), Cx_(chunk_position.x), Cy_(chunk_position.y), Cz_(chunk_position.z) {
-    for (Dim x = SIZE * Cx_; x < SIZE * (1 + Cx_); x++)
-        for (Dim y = SIZE * Cy_; y < SIZE * (1 + Cy_); y++)
-            for (Dim z = SIZE * Cz_; z < SIZE * (1 + Cz_); z++) {
-                if (ter_->can_stand_1(x, y, z)) {
+    ter_(ter), chunk_position_(chunk_position),
+    tiles_(SIZE * SIZE * SIZE, Tile(ter_->get_material(0), 0)) {}
+
+// initialize all tiles. this is done above. currently it is wrong.
```

Comment:
Fix it then lol

Reply:
I think I just forgot to remove the comment. What if above?? This is line 14